hapi-loader/application.docker.yaml

```diff
@@ -35,6 +33,12 @@ spring:
   batch:
     job:
       enabled: false
+  elasticsearch:
```

Comment:
Why do we need that now?

Reply:
because when you do a healthcheck on the `/actuator/health` endpoint, micrometer/spring is smart enough to make sure that the connection to ES is healthy, therefore you need to provide credentials in the spring configuration